src/components/Layout.tsx

```diff
@@ -180,7 +165,7 @@ const Layout: React.FC<IProps> = ({
                 <Nav path={path} />
                 {shouldShowSideNav && <SideNavMobile path={path} />}
               </VisuallyHidden>
-              <BannerNotification
+              {/* <BannerNotification
```

Comment:
I suspect we should remove the `"merge-complete"` string from common.json?

Reply:
Oops, good catch. Removing string and comment now.